build.gradle.kts

```diff
@@ -11,9 +11,9 @@ import org.gradle.api.tasks.testing.logging.TestExceptionFormat
  */
 // project version
 // pom artifact version used when the built artifact is published
-version = "2.2.0"
+version = "2.3.0"
 // Test jar library version used in the task 'generateTestJar'
-val release = "2.2.0"
+val release = "2.3.0"
```

Comment:
Something to investigate in the future, no idea why we have 2 values which should mean the same thing.